### PR TITLE
Create config

### PR DESCRIPTION
**Pull request by dmahfouda**
Hi Mikey, I pulled out the environment variables! Take a look!

### CODE REVIEW

File: tin-can-phone.ino [hunk 1]
```diff
@@ -401,12 +403,8 @@ void readSerial () {
...
 void setup() {
     USE_SERIAL.begin(115200);
-    //can.connectToWifi("Recurse Center", "nevergraduate!");
-    //can.connectToWifi("MySpectrumWiFib8-2G", "classypoodle861");
-    can.connectToWifi("s&m", "passwordispassword");
-    //can.openWebSocket("10.0.20.109", 3000); Recurse
-    //can.openWebSocket("192.168.1.18", 3000); //David's House
-    can.openWebSocket("192.168.1.2", 3000); //Mikey's House
+    can.connectToWifi(TIN_CAN_SSID, TIN_CAN_PASSWORD);
+    can.openWebSocket(TIN_CAN_SWITCH, 3000);
```
Comment (mgoldenberg) [inline tin-can-phone.ino]: Hm, I forgot about the port. Let's add another configuration constant for that and change the name of `TIN_CAN_SWITCH` to represent the host (i.e. server name or ip). In other words, we'd have:

- `TIN_CAN_SWITCH_HOST`
- `TIN_CAN_SWITCH_PORT`

File: tin-can-phone.ino [hunk 2]
```diff
@@ -196,7 +197,8 @@ class Can {
 
         void openWebSocket(String ip, int port) {
             this->webSocket.beginSocketIO(ip, port);
-            //webSocket.setAuthorization("user", "Password"); // HTTP Basic Authorization
+            String header_string = String("name:") + TIN_CAN_ID;
+            this->webSocket.setExtraHeaders(header_string.c_str());
             this->webSocket.onEvent(webSocketEvent);
         }
```
Comment (mgoldenberg) [inline tin-can-phone.ino:204]: On line 269, below here, you should build the URL with the constants defined in the configuration file.

**Review by mgoldenberg — CHANGES_REQUESTED** [review]
This looks good! I think just a few small changes and we're ready to merge, see the comments in the files.

File: tin-can-phone.ino (hunk 2 above)
Comment (mgoldenberg) [reply tin-can-phone.ino:204]: (You can't see the line I'm referring to in this snippet, by the way. For some reason I couldn't comment on a part of the code that hasn't changed.)

**Comment by dmahfouda** [comment]
Hi @mgoldenberg take another look. I'm not able to see the websocket with this version, but all of the config variables are now compiling. This should resolve #3

**Comment by mgoldenberg** [comment]
Looks great!

**Review by mgoldenberg — APPROVED** [review]

**Pull request merged**